### DISCUSSION TITLE
考虑增加4o-Latest吗？

### DISCUSSION THREAD

**Discussion by YersiniaHerb** [BurnHair/docs#205] [功能建议]
openai上面已经有4o-Latest模型了，考虑增加吗？

**Comment by BennyThink** [comment]
https://api.burn.hair/ 这里有
gpt-4o-2024-08-06,gpt-4o,dall-e-2,whisper-1,gpt-3.5-turbo-instruct,tts-1-hd-1106,gpt-4o-mini-2024-07-18,gpt-4o-mini,gpt-3.5-turbo,gpt-3.5-turbo-0125,babbage-002,davinci-002,dall-e-3,tts-1,gpt-3.5-turbo-16k,tts-1-1106,chatgpt-4o-latest,gpt-4-turbo-2024-04-09,gpt-4,tts-1-hd,gpt-4o-realtime-preview-2024-10-01,gpt-4o-realtime-preview,gpt-4-turbo-preview,gpt-4-1106-preview,text-embedding-ada-002,gpt-4-turbo,text-embedding-3-small,text-embedding-3-large,gpt-3.5-turbo-1106,gpt-4o-audio-preview,gpt-4o-audio-preview-2024-10-01,gpt-4-0613,gpt-4-0125-preview,gpt-3.5-turbo-instruct-0914,gpt-4o-2024-05-13
